ios/BetterSocial/BetterSocial.entitlements

```diff
@@ -8,9 +8,11 @@
 	<array>
 		<string>applinks:link.bettersocial.org/users</string>
 		<string>applinks:bettersocial.page.link/users</string>
-		<string>applinks:linkdev.bettersocial.org/posts</string>
-		<string>applinks:linkdev.bettersocial.org/postprivate</string>
-		<string>applinks:linkdev.bettersocial.org/postexpired</string>
+		<string>applinks:link.bettersocial.org/posts</string>
+		<string>applinks:link.bettersocial.org/postprivate</string>
+		<string>applinks:link.bettersocial.org/postexpired</string>
+		<string>applinks:helio.social</string>
+		<string>applinks:app.bettersocial.org</string>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new entitlements `applinks:helio.social` and `applinks:app.bettersocial.org` have been added. Ensure that these domains are correctly configured in your Apple App Site Association file and they are owned by you or you have permissions to use them. Misconfiguration can lead to Universal Links not working as expected.
```diff
+		<string>applinks:helio.social</string>
+		<string>applinks:app.bettersocial.org</string>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->